### PR TITLE
Fix update of sibling-count bug

### PR DESCRIPTION
**Pull request by 88manpreet**
In the test case: Before the fix the net_imbal is 3, the original imbal is 5

### CODE REVIEW

File: kafka_utils/kafka_cluster_manager/cluster_info/rg.py [hunk 1]
```diff
@@ -243,7 +243,7 @@ def rebalance_brokers(self):
                 sibling_distance = self.update_sibling_distance(
                     sibling_distance,
                     broker_destination,
-                    victim_partition,
+                    victim_partition.topic,
```
Comment (ecanzonieri) [inline kafka_utils/kafka_cluster_manager/cluster_info/rg.py]: Nice catch!

File: tests/kafka_cluster_manager/cluster_topology_test.py [hunk 2]
```diff
@@ -236,6 +238,34 @@ def test_rebalance_replication_groups(self):
             ct.brokers.keys(),
         )
 
+    def test_rebalance_brokers_for_topic_partition_imbalance(self):
```
Comment (ecanzonieri) [inline tests/kafka_cluster_manager/cluster_topology_test.py]: I think we should add a unit test in rg_test.py rather than in ClusterTopology.

**Comment by 88manpreet** [comment]
Not able to reply to your comment.
This test builds cluster-toplogy and rebalance all brokers. So I guess its better here?

**Comment by 88manpreet** [comment]
Not able to reply to your comment.
This test builds cluster-toplogy and rebalance all brokers. So I guess its better here?

**Comment by 88manpreet** [comment]
Not able to reply to your comment.
This test builds cluster-toplogy and rebalance all brokers. So I guess its better here?

File: tests/kafka_cluster_manager/cluster_topology_test.py (hunk 2 above)
Comment (88manpreet) [inline tests/kafka_cluster_manager/cluster_topology_test.py]: Not able to reply to your comment.
This test builds cluster-toplogy and rebalance all brokers. So I guess its better here?

**Pull request merged**